runisort

```diff
@@ -1,3 +1,2 @@
 #!/bin/bash
-set -e
-isort --check-only --recursive --diff --skip migrations || exit 1
+isort --check-only --recursive --diff .
```

Comment:
add the new line at the end of the file and configure your editor to always add it